### ISSUE TITLE
Not entirely clear example "Use the same vocabulary for the same type of variable"

### ISSUE THREAD

**Issue by matiit** [jupeter/clean-code-php#92]
https://github.com/jupeter/clean-code-php#use-the-same-vocabulary-for-the-same-type-of-variable

It's hard to quickly tell what is the subject of the issue here.
I don't see problem in class having methods like: 
```php
getUserInfo();
getClientData();
```

Because User and Client can be actually different entities. 

I think it would be good to provide more description explaining that it refers to calling same Entity with different names, which should generally be avoided?

**Comment by meadsteve** [comment]
Maybe the example would be better like:

**Bad:**
```php
getUserInfo();
getUserData();
```

**Good:**
```php
getUser();
```

**Comment by peter-gribanov** [comment]
Thanks. I created PR #93 for this.

**Comment by TomasVotruba** [comment]
Resolved by #93
